string relay
int16 delay
int16 duration
---
string status